public/content/roadmap/fusaka/index.md

```diff
@@ -192,6 +192,83 @@ Yes, the Fusaka upgrade requires updates to both [execution clients and consensu
+
+### What EVM changes are implemented? {#what-evm-changes-are-implemented}
+
+Fusaka solidifies EVM with new minor changes and features. 
+
+- For security while scaling, a maximum size of a single transaction will be [limited to 16.7 million](https://eips.ethereum.org/EIPS/eip-7825) units of gas. 
+- [New opcode CLZ](https://eips.ethereum.org/EIPS/eip-7939), Count Leading Zeros, is added to EVM and will enable smart contract languages to perform certain operations more efficiently.
+- [The cost ModExp precompile will be increased](https://eips.ethereum.org/EIPS/eip-7883), contracts using it will charge more gas for execution.
+
+### How does new 16M gas limit affects contract developers? {#how-does-new-16m-gas-limit-affects-contract-developers}
+
+Fusaka introduces a limit to [maximum size of a single transaction to 16.7 million](https://eips.ethereum.org/EIPS/eip-7825) (2^24) gas units. This is roughly the size of an average block before which makes it big enough to accommodate even complex transactions that would consume an entire block. This limit creates protection for clients, preventing potential DoS attacks in the future with higher block gas limit. The goal of scaling is to enable more transactions get into the blockchain without a single one consuming all the block. 
+
+Regular user transactions are far from reaching this limit. Certain edge cases like big and complex Defi operations, large smart contract deployements or batch transactions targeting multiple contracts might be affected by this change. These transaction will have to be divided into smaller ones or optimized in another way. Use simulation before submitting transactions that potentially reach the limit.
+
+RPC method `eth_call` is not limited and will allow simulation of bigger transactions than the actual blockchain limit. The actual limit for RPC methods can be configured by the client operator to ensure prevent abuse. 
+
+### What CLZ means for developers? {#what-clz-means-for-developers}
+
+EVM compilers like Solidity will implement and utilize the new function for counting zeroes under the hood. New contracts might benefit from gas savings if they rely on this sort of operations. Follow releases and feature announcement of the smart contract language for documentation on potential savings. 
+
+### Are there any changes for my existing smart contracts? {#what-clz-means-for-developers}
+
+Fusaka has no direct affect that would break any existing contracts or change their behavior. Changes introduced to execution layer are made with backwards compatibility, however, always keep an eye on edge cases and potential impact. 
```

Comment:
```suggestion
Fusaka has no direct affect that would break any existing contracts or change their behavior. Changes introduced to the execution layer are made with backward compatibility, however, always keep an eye on edge cases and potential impact. 
```